contracts/RMRK/library/RMRKErrors.sol

```diff
@@ -2,59 +2,124 @@
 error RMRKMaxPendingChildrenReached();
+/// Attempting to add a pending resource after the number of pending resources has reached the limit (default limit is
+///  128)
 error RMRKMaxPendingResourcesReached();
+/// Attempting to mint a number of tokens that would cause the total supply to be greater than maximum supply
 error RMRKMintOverMax();
+/// Attempting to mint a nested token to a smart contract that doesn't support nesting
 error RMRKMintToNonRMRKImplementer();
+/// Attempting to unnest a child before it is unequipped
 error RMRKMustUnequipFirst();
+/// Attempting to nest a child over the nesting limit (current limit is 100 levels of nesting)
 error RMRKNestingTooDeep();
+/// Attempting to nest the token to own descendant, which would create a loop and leave the looped tokens in limbo
 error RMRKNestingTransferToDescendant();
+/// Attempting to nest the token to a smart contract that doesn't support nesting
 error RMRKNestingTransferToNonRMRKNestingImplementer();
+/// Attempting to nest the token into itself
 error RMRKNestingTransferToSelf();
+/// Attempting to interact with a resource that can not be found
 error RMRKNoResourceMatchingId();
+/// Attempting to manage a resource without owning it or having been granted permission by the owner to do so
 error RMRKNotApprovedForResourcesOrOwner();
+/// Attempting to interact with a token or resource without being its owner or having been granted permission by the
+///  owner to do so
 error RMRKNotApprovedOrDirectOwner();
+/// Attempting to compose a resource wihtout an address
 error RMRKNotComposableResource();
+/// Attempting to unequip an item that isn't equipped
 error RMRKNotEquipped();
+/// Attempting to interact with a token or resource without being its owner
 error RMRKNotOwner();
+/// Attempting to interact with a function without being the owner or contributor of the collection
 error RMRKNotOwnerOrContributor();
+/// Attempting to transfer the ownership to the 0x0 address
 error RMRKNewOwnerIsZeroAddress();
+/// Attempting to assign a 0x0 address as a contributor
 error RMRKNewContributorIsZeroAddress();
+/// Attempting to add a `Part` with an ID that is already used
 error RMRKPartAlreadyExists();
+/// Attempting to use a `Part` that doesn't exist
 error RMRKPartDoesNotExist();
+/// Attempting to use a `Part` that is `Fixed` when `Slot` kind of `Part` should be used
 error RMRKPartIsNotSlot();
+/// Attempting to interact with a pending child using an index greater than the size of pending array
 error RMRKPendingChildIndexOutOfRange();
+/// Attempting to add a resource using an ID that has already been used
 error RMRKResourceAlreadyExists();
+/// Attempting to equip an item into a slot that already has an item euipped
 error RMRKSlotAlreadyUsed();
+/// Attempting to equip an item into a resource that is not a `Slot`
 error RMRKTargetResourceCannotReceiveSlot();
+/// Attempting to equip a child into a slot that doesn't support the tokens from child's collection
```

Comment:
/// Attempting to equip a child into a slot and parent that the child does not support.